### DISCUSSION TITLE
开始抢票后报错

### DISCUSSION THREAD

**Discussion by Longinusli** [mikumifa/biliTickerBuy#148] [提问专区]
![image](https://github.com/mikumifa/biliTickerBuy/assets/43884830/48cbe104-2bb9-4dd0-9d44-567118d36053)

如图，请问可能是什么问题？（抢的票需要舰长）

**Comment by mikumifa** [comment]
把过程日志都发一下吧，敏感信息删掉 （app.log文件）

**Reply by Longinusli** [reply]
[app.log](https://github.com/user-attachments/files/15926611/app.log)
Thanks♪(･ω･)ﾉ

**Comment by mikumifa** [comment]
连续访问，正常都会这样的，对面票可能没发售

**Reply by Longinusli** [reply]
好的，感谢
